pay-api/migrations/versions/2023_02_23_0a3ad7c041f6_add_refund_date_to_invoices.py

```diff
@@ -0,0 +1,24 @@
+"""add refund date to invoices
+
+Revision ID: 0a3ad7c041f6
+Revises: e296910623cd
+Create Date: 2023-02-23 09:49:57.642039
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0a3ad7c041f6'
+down_revision = 'e296910623cd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('invoices', sa.Column('refund_date', sa.DateTime(), nullable=True))
```

Comment:
please add `op.execute("set statement_timeout=20000;")` before this update to invoices

Reply:
Yes, I forgot this.. if we don't add this statement it can cause a deadlock due to so many incoming connections while trying to get an exclusive lock on the table.

Reply:
ok, just added. thanks!